Subsurface/Node/Common/Peripherals/Src/DPS310.cpp

```diff
@@ -0,0 +1,771 @@
+#include "Peripherals/Inc/DPS310.h"
```

Comment:
@Ian-Carroll do you wanna just comb through this and comment the functions that you changed? I only wanna review the stuff that you modified and not any of the stuff that you didn't

Reply:
@Burke-Daniel I took note of all the functions changed through the commits in this PR, and they go as follows:

- configTemp
- configPressure
- getFIFOvalue
- begin
- readcoeffs
- readBlock
- readByte
- writeByte
- readByteBitField
- measureTempOnce
- measurePressureOnce
- disableFIFO
- getFIFOvalue
- getRawResult
- HAL_I2C_Mem_Read on line 36 of I2CInterface.cpp